storage/src/adb/any/fixed/mod.rs

```diff
@@ -109,11 +108,11 @@ pub struct Any<
     /// # Invariant
     ///
     /// Only references operations of type [Operation::Update].
-    pub(crate) snapshot: Index<T, u64>,
+    pub(crate) snapshot: Index<T, Location>,
 
     /// A location before which all operations are "inactive" (that is, operations before this point
     /// are over keys that have been updated by some operation at or after this point).
-    pub(crate) inactivity_floor_loc: u64,
+    pub(crate) inactivity_floor_loc: Location,
```

Comment:
nit: inactivity_floor

Reply:
I'm going to leave this for the follow-up PR where I change the return types of `mmr.size()` and `[adb].op_count()` if that's alright